/**:
  ros__parameters:
    racing_mpc:
      max_cpu_time: 0.2
      max_iter: 30
      tol: 0.1
      constr_viol_tol: 0.1
      n: 40
      margin: 0.3
      average_track_width: 11.0
      verbose: false

      q_contour: 0.1
      q_progress: 0.02
      r: [
        1.0, 0.0, 0.0,
        0.0, 1.0, 0.0,
        0.0, 0.0, 1.0
      ]

      x_max: [.inf, .inf, .inf, 1.5, 0.5236, 100.0]
      x_min: [-.inf, -.inf, -.inf, -1.5, -0.5236, 0.0]
      u_max: [7100.0, 0.0, 0.2513]
      u_min: [0.0, -20000.0, -0.2513]
